src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -571,18 +574,40 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_ep_t, uct_iface_h tl_iface)
     return UCS_OK;
 }
 
+static void uct_rc_mlx5_ep_reset_qp(uct_rc_mlx5_ep_t *ep)
```

Comment:
do we need to call it for TM qp as well?

Reply:
yes, currently there is not SRQ cleanup flow for TM QP at all..

Reply:
since it requires major change i prefer to do it in separate PR, and merge this one to make tests pass